react/src/components/base/Typography/Typography.stories.js

```diff
@@ -13,7 +13,7 @@ import {
   styles
 } from './typography.json';
 import typographyOptions from './Typography.knobs.options';
-import TypographyDocs from '../../../../../docs/style/typography.md';
+import TypographyDocs from './typography.md';
```

Comment:
Is there a reason to duplicate this doc from the [gitbook subrepo](/docs)

Reply:
Currently, the docs sub repo is syncing from Gitbook, which allows non-tech people to use the wysiwyg interface to collaborate on documentation